README.md

```diff
@@ -1,53 +1,109 @@
-# Wiki
+# Publish to GitHub wiki action
 
-This action takes the contents of a provided directory and synchronizes it with your
-repository's wiki. This allows, for example, pull requests for the wiki.
+📖 Deploy docs from your source tree to the GitHub wiki
 
-The goal is to expose the wiki to PRs, so that potential contributors who aren't
-collaborators can contribute to the wiki and go through a review process. Alternatively,
-one can disabled the "Restrict editing to collaborators only" setting for their wiki,
-but this allows the community to make unreviewed, potentially unwanted, changes.
+<div align="center">
 
-By default, it will use the `.github/wiki` directory.
+![](https://placekitten.com/600/400)
```

Comment:
Why are we adding a picture of a cat? :laughing:

Reply:
Currently, it's a draft pr, and I didn't have a good header image. Here's the previous one that I used for my own wiki action, but it needs some editing:

![image](https://user-images.githubusercontent.com/61068799/225371964-8e685085-6092-4792-ab70-aaa55956768d.png)

So I just put a cat placeholder "placekitten" for now at 600x400 which is my usual target size for readme headers. Not too big, but not too small.

Reply:
Frankly, I think we can omit the header image for this one. It's certainly eye-catching, but it requires vertical scrolling to get to relevant information. We should consider what this "product" is and who the potential users are. This is not a big project with a large ecosystem. Our users *must* be GitHub users with at least the general coding skills needed to write a workflow. Because of this, I think that boring, accessible info takes priority over fun images. While pictures may say 1000 words, are they words that haven't already been said? I think the repo description that you suggested does a wonderful job of saying what needs to be said :slightly_smiling_face: 

My personal opinion as a reader: when I'm looking to use an action, I just want to know what it does, copy and paste some stuff, and get started. The less scrolling I need to know what I need to know, the better.

Reply:
Sounds good! 👍

Reply:
Do you prefer this:

![image](https://user-images.githubusercontent.com/61068799/225409945-63c99d23-6f25-4287-a8f7-2ec0864dcd02.png)

Or this?

![image](https://user-images.githubusercontent.com/61068799/225409426-eeffdcc4-b354-4dfe-99f1-cd1dac1e07f1.png)

Reply:
@spenserblack still need to know if I should keep the "quicknav" thing or if it looks too out-of-place without the image (⚠ it does use `<div align=center>` 😅)

Reply:
Those links are definitely handy, but maybe they can be addressed in a separate PR.

Reply:
Fixed in https://github.com/jcbhmr/actions-wiki/commit/4992f0abb0681d9b30d9a0877e671445fed1138e